snap7/server.py

```diff
@@ -242,7 +243,8 @@ def set_mask(self, kind: int, mask: int):
     def set_cpu_status(self, status: int):
         """Sets the Virtual CPU status.
         """
-        assert status in snap7.types.cpu_statuses, f'unknown cpu state {status}'
+        if status not in snap7.types.cpu_statuses:
+            raise KeyError(f"unknown cpu state {status}")
```

Comment:
value error

Reply:
cpu_statuses is a dict, so I guess it is implicit done anyway